edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -243,7 +243,7 @@ where
 
         let file_name = format!("{}_log.txt", module_name);
         zip_writer
-            .start_file_from_path(&Path::new("logs").join(file_name), file_options)
+            .start_file(Path::new("logs").join(file_name).to_string_lossy(), file_options)
```

Comment:
Need to use `/` even on Windows, based on what `start_file_from_path` was doing.

https://docs.rs/zip/0.5.13/src/zip/write.rs.html#400-401

https://docs.rs/zip/0.5.13/src/zip/write.rs.html#1166

Reply:
I overlooked the `push('/')`, thank you for the catch. That is a major potential footgun, in any case.

Reply:
Module names were space-free, right?

Reply:
Yeah. All the callers call this with regular `String` components anyway, so you can just `format!("logs/{}", file_name)` etc it instead of involving `Path`